src/core/execute.c

```diff
@@ -3448,8 +3448,12 @@ static int exec_child(
         if (context->protect_hostname) {
```

Comment:
commit message: PrivateHostname= -> ProtectHostname=

Reply:
indeed! thanks!

Reply:
(and fixed)